packages/commonwealth/server/models/thread.ts

```diff
@@ -8,6 +8,18 @@ import type { LinkedThreadAttributes } from './linked_thread';
 import type { TopicAttributes } from './topic';
 import type { ModelInstance, ModelStatic } from './types';
 
+export enum linkSource {
```

Comment:
`linkSource` => `LinkSource`
look like this is our convention

Reply:
fixed this